src/test/rgw/rgw_multi/tests.py

```diff
@@ -957,9 +962,18 @@ def test_datalog_autotrim():
 
     # trim each datalog
     for zone, _ in zone_bucket:
+        # read max markers for each shard
+        status = datalog_status(zone.zone)
+
         datalog_autotrim(zone.zone)
-        datalog = datalog_list(zone.zone)
-        assert len(datalog) == 0
+        for shard_id, shard_status in enumerate(status):
+            max_marker = shard_status['marker']
+            if not len(max_marker):
+                continue
+            entries = datalog_list(zone.zone, ['--shard-id', str(shard_id)])
+            if not len(entries):
+                continue
+            assert(max_marker < entries[0]) # datalog entries must be newer
```

Comment:
you check only entries[0] because they are ordered?

Reply:
yes this in an omap listing and the markers are the keys, so they're guaranteed to be in sorted order

Reply:
oops, this isn't right. entries[0] is a dict that looks like:
```json
    {                                                                                                                                                         
        "entity_type": "bucket",                                                                                                                              
        "key": "lrpwav-10:6c4646ef-65e9-4875-8be3-61e59fc0e2cd.4149.10:0",                                                                                    
        "timestamp": "2020-08-25T11:54:43.637049Z"                                                                                                            
    },
```
and we can't even see the markers. i guess i'll have to compare that timestamp with the timestamp from `radosgw-admin datalog status`:
```json
    {
        "marker": "1_1598356500.981383_31.3",
        "last_update": "2020-08-25T11:55:00.981383Z"
    },
```